Dockerfile

```diff
@@ -1,23 +1,24 @@
-FROM node:6.9.5 as app
+FROM node:10.17.0-jessie as build
```

Comment:
I think Dockerfile comes from the build scripts, so is it going to get overwritten on the next update?

Reply:
Oh, right. I've not really made friends with the build scripts.

Looks like the optimisations I've made won't work in that case.

Actually, it looks like the s3-related things that I put in the other day will get overwritten too.